### PR TITLE
Updated team links

### PR DESCRIPTION
**Pull request by ekuiter**
Some links were outdated.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -545,9 +545,9 @@ contact the <a href="mailto:featureide@uni-ulm.de">FeatureIDE development team</
 <li>Lukas Vogel (University of Magdeburg, Germany)</li>
 <li>Maik Lampe (University of Magdeburg, Germany)</li>
 <li>Malek Badeer (TU Braunschweig, Germany)</li>
-<li><a href="https://www.ict.swin.edu.au/personal/mkapuruge/">Malinda Kaushalye Kapuruge</a> (Swinburne University of Technology, Melbourne, Australia)</li>
+<li>Malinda Kaushalye Kapuruge (Swinburne University of Technology, Melbourne, Australia)</li>
 <li>Marcel Daniel (TU Braunschweig, Germany)</li>
-<li><a href="http://www.ic.ufal.br/marcio">M&aacute;rcio Ribeiro</a> (Federal University of Alagoas, Brazil)</li>
+<li>M&aacute;rcio Ribeiro (Federal University of Alagoas, Brazil)</li>
```
Comment (tthuem) [inline index.md:550]: This links works for me and it forwards to: https://sites.google.com/a/ic.ufal.br/marcio/

File: index.md [hunk 2]
```diff
@@ -568,9 +568,9 @@ contact the <a href="mailto:featureide@uni-ulm.de">FeatureIDE development team</
 <li>Patrik Haese (University of Magdeburg, Germany)</li>
 <li>Philipp Kuhn (TU Braunschweig, Germany)</li>
 <li>Philipp Vulpius (University of Ulm, Germany)</li>
-<li><a href="https://wwwiti.cs.uni-magdeburg.de/~rschroet/">Reimar Schr&ouml;ter</a> (University of Magdeburg, Germany)</li>
+<li>Reimar Schr&ouml;ter (University of Magdeburg, Germany)</li>
```
Comment (tthuem) [inline index.md:571]: This is unfortunate that you put websites offline in Magdeburg. Please at least link to former members if the page does not exist anymore.

File: index.md [hunk 3]
```diff
@@ -568,9 +568,9 @@ contact the <a href="mailto:featureide@uni-ulm.de">FeatureIDE development team</
...
 <li>Sabrina Hugo (TU Braunschweig, Germany)</li>
-<li><a href="https://www.tu-braunschweig.de/isf/team/schulze">Sandro Schulze</a> (University of Magdeburg, Germany)</li>
+<li><a href="https://www.dbse.ovgu.de/Mitarbeiter/Assoziierte+Mitarbeiter/Sandro+Schulze.html">Sandro Schulze</a> (University of Magdeburg, Germany)</li>
```
Comment (tthuem) [inline index.md:573]: Is currently in Potsdam: https://www.uni-potsdam.de/en/cs-se/team

**Review by tthuem — CHANGES_REQUESTED** [review]
We are almost there.

**Comment by ekuiter** [comment]
I made some changes (and added some more links) but somehow the commits don't show up here although I pushed to patch-1 :-(

**Comment by tthuem** [comment]
Trying to create a new PR.

**Pull request closed without merging**